.github/workflows/array_api.yml

```diff
@@ -15,24 +15,8 @@ permissions:
 env:
   CCACHE_DIR: "${{ github.workspace }}/.ccache"
   INSTALLDIR: "build-install"
-  XP_TESTS: >-
-    -t scipy.cluster
-    -t scipy.constants
-    -t scipy.fft
-    -t scipy.special.tests.test_logsumexp
-    -t scipy.special.tests.test_support_alternative_backends
-    -t scipy._lib.tests.test_array_api
-    -t scipy._lib.tests.test__util
-    -t scipy.differentiate.tests.test_differentiate
-    -t scipy.integrate.tests.test_tanhsinh
-    -t scipy.integrate.tests.test_cubature
-    -t scipy.optimize.tests.test_bracket
-    -t scipy.optimize.tests.test_chandrupatla
-    -t scipy.optimize.tests.test_optimize
-    -t scipy.stats
-    -t scipy.ndimage
-    -t scipy.integrate.tests.test_quadrature
-    -t scipy.signal.tests.test_signaltools
+  # Only run tests that use the `xp` fixture
+  XP_TESTS: -m array_api_backends
```

Comment:
Are there any tests that don't use the `xp` parameter that would be important to run with `SCIPY_ARRAY_API=1`?

Reply:
probably fine to just run those with `xp` for now. We can always expand coverage in the future.